### PR TITLE
Doc samples for Self Service Maintenance feature for Cloud SQL

### PR DESCRIPTION
**Pull request by 2tricpony**
Doc samples for Self Service Maintenance feature for Cloud SQL

### CODE REVIEW

File: sql_sqlserver_instance_ssm/test.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: blueprints.cloud.google.com/v1alpha1
+kind: BlueprintTest
+metadata:
+  name: sql_mysql_instance_pvp
+spec:
+  skip: true
```
Comment (bharathkkb) [inline sql_sqlserver_instance_ssm/test.yaml:20]: nit: add newline

File: sql_sqlserver_instance_ssm/test.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: blueprints.cloud.google.com/v1alpha1
+kind: BlueprintTest
+metadata:
+  name: sql_mysql_instance_pvp
```
Comment (bharathkkb) [inline sql_sqlserver_instance_ssm/test.yaml:18]:
```suggestion
  name: sql_sqlserver_instance_ssm
```

File: sql_postgres_instance_ssm/test.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: blueprints.cloud.google.com/v1alpha1
+kind: BlueprintTest
+metadata:
+  name: sql_mysql_instance_pvp
```
Comment (bharathkkb) [inline sql_postgres_instance_ssm/test.yaml:18]:
```suggestion
  name: sql_postgres_instance_ssm
```

File: sql_mysql_instance_ssm/test.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: blueprints.cloud.google.com/v1alpha1
+kind: BlueprintTest
+metadata:
+  name: sql_mysql_instance_pvp
```
Comment (bharathkkb) [inline sql_mysql_instance_ssm/test.yaml:18]:
```suggestion
  name: sql_mysql_instance_ssm
```

File: sql_sqlserver_instance_ssm/main.tf [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+  name                = "sqlserver-ssm-instance-name"
+  region              = "asia-northeast1"
+  database_version    = "SQLSERVER_2019_STANDARD"
+  maintenance_version = "SQLSERVER_2019_STANDARD_CU16_GDR.R20220821.00_00"
+  settings {
+    tier              = "db-f1-micro"
+  }
+  deletion_protection = false # set to true to prevent destruction of the resource
+}
+# [END cloud_sql_sqlserver_instance_ssm]
```
Comment (bharathkkb) [inline sql_sqlserver_instance_ssm/main.tf:12]: nit: add newline

File: sql_sqlserver_instance_ssm/test.yaml (hunk 2 above)
Comment (bharathkkb) [reply sql_sqlserver_instance_ssm/test.yaml:18]: The name should be the name of the sample dir

File: sql_sqlserver_instance_ssm/main.tf (hunk 5 above)
Comment (2tricpony) [reply sql_sqlserver_instance_ssm/main.tf]: Done

File: sql_sqlserver_instance_ssm/test.yaml (hunk 1 above)
Comment (2tricpony) [reply sql_sqlserver_instance_ssm/test.yaml:20]: Done.

**Comment by 2tricpony** [comment]
Done. Please take another look.

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**